src/window/mod.rs

```diff
@@ -52,7 +56,7 @@ pub enum WindowCommand {
     ListAvailableFonts,
 }
 
-pub fn create_window() {
+pub fn create_window() -> ExitCode {
```

Comment:
Could it be cleaner to instead return an error enum in `Result<Infallible, ...>`?

Reply:
I don't think so, the function can return both Ok and an Error, we could move the error code translation up to main, and perhaps extract it to a function though.

The `create_window` function should perhaps be named to `create_window_and_run` to indicate that it does more than just creates the window. Or perhaps split into two separate functions, one that creates the event loop (which owns everything it needs) and one that runs it.